modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/jdbc/JdbcStatementSelfTest.java

```diff
@@ -0,0 +1,874 @@
+            stmt.close();
+
+            assertTrue(stmt.isClosed());
+        }
+
+        conn.close();
+
+        assertTrue(stmt.isClosed());
+        assertTrue(conn.isClosed());
+    }
+
+    /**
+     * TODO IGNITE-15187 + IGNITE-15108
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    @Disabled
+    public void testExecuteQuery0() throws Exception {
+        ResultSet rs = stmt.executeQuery(SQL);
+
+        assertNotNull(rs);
+
+        int cnt = 0;
+
+        while (rs.next()) {
+            int id = rs.getInt("id");
+
+            if (id == 2) {
+                assertEquals("Joe", rs.getString("firstName"));
+                assertEquals("Black", rs.getString("lastName"));
+                assertEquals(35, rs.getInt("age"));
+            }
+            else if (id == 3) {
+                assertEquals("Mike", rs.getString("firstName"));
+                assertEquals("Green", rs.getString("lastName"));
+                assertEquals(40, rs.getInt("age"));
+            }
+            else
+                fail("Wrong ID: " + id);
+
+            cnt++;
+        }
+
+        assertEquals(2, cnt);
+    }
+
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testExecuteQuery1() throws Exception {
+        final String sqlText = "select 5;";
+
+        try (ResultSet rs = stmt.executeQuery(sqlText)) {
+            assertNotNull(rs);
+
+            assertTrue(rs.next());
+
+            int val = rs.getInt(1);
+
+            assertTrue(val >= 1 && val <= 10, "Invalid val: " + val);
+        }
+
+        stmt.close();
+
+        // Call on a closed statement
+        checkStatementClosed(() -> stmt.executeQuery(sqlText));
+    }
+
+    /**
+     * TODO IGNITE-15187 + IGNITE-15108
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    @Disabled
+    public void testExecute() throws Exception {
+        assertTrue(stmt.execute(SQL));
+
+        assertEquals(-1, stmt.getUpdateCount(), "Update count must be -1 for SELECT query");
+
+        ResultSet rs = stmt.getResultSet();
+
+        assertNotNull(rs);
+
+        int cnt = 0;
+
+        while (rs.next()) {
+            int id = rs.getInt("id");
+
+            if (id == 2) {
+                assertEquals("Joe", rs.getString("firstName"));
+                assertEquals("Black", rs.getString("lastName"));
+                assertEquals(35, rs.getInt("age"));
+            }
+            else if (id == 3) {
+                assertEquals( "Mike", rs.getString("firstName"));
+                assertEquals( "Green", rs.getString("lastName"));
+                assertEquals(40, rs.getInt("age"));
+            }
+            else
+                fail("Wrong ID: " + id);
+
+            cnt++;
+        }
+
+        assertEquals(2, cnt);
+
+        assertFalse(stmt.getMoreResults(), "Statement has more results.");
+    }
+
+    /**
+     * TODO IGNITE-15187 + IGNITE-15108
+     * @throws Exception If failed.
+     */
+    @Test
+    @Disabled
```

Comment:
```suggestion
     * @throws Exception If failed.
     */
    @Test
    @Disabled("IGNITE-15187 + IGNITE-15108")
```

Reply:
This is more informative and allow us to see the reason on TC in logs and reports.